sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -1023,16 +1029,20 @@
                 </configuration>
               </execution>
 
-              <!-- compile tests with Java 11 -->
-              <execution>
-                <id>base-modules-testCompile</id>
-                <goals>
-                  <goal>testCompile</goal>
-                </goals>
-                <configuration>
-                  <testRelease>11</testRelease>
-                </configuration>
-              </execution>
+              <!-- compile tests with Java 9 -->
+              <!-- At this time recompiling tests with Java 9 isn't necessary as they don't have module-infos. If/when they have module-infos added uncomment this execution. -->
+<!--              <execution>-->
+<!--                <id>base-modules-testCompile</id>-->
+<!--                <goals>-->
+<!--                  <goal>testCompile</goal>-->
+<!--                </goals>-->
+<!--                <configuration>-->
+<!--                  <testRelease>9</testRelease>-->
+<!--                  <testIncludes>-->
+<!--                    <testInclude>module-info.java</testInclude>-->
+<!--                  </testIncludes>-->
+<!--                </configuration>-->
+<!--              </execution>-->
```

Comment:
We can either
remove this and make a note in https://github.com/Azure/azure-sdk-for-java/issues/29245 .
Or keep it and reference https://github.com/Azure/azure-sdk-for-java/issues/29245 in the comments.